src/pages/api/experimental/stats/index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Stats API
```

Comment:
This is a customer-facing page and link, so we should be formal here and call this something like Statistics.

Reply:
Updated it to Statistics